probe/endpoint/ebpf.go

```diff
@@ -287,10 +310,33 @@ func (t *EbpfTracker) isDead() bool {
 
 func (t *EbpfTracker) stop() {
 	t.Lock()
+	defer t.Unlock()
+
```

Comment:
I deliberately didn't do this, because it requires a lot of assumptions about the behaviour of tracer, in order to avoid deadlocks. It's fine right now.

Was there a specific reason why you changed this?